src/pages/registration/contributors_tab/AddContributorModal.tsx

```diff
@@ -53,9 +54,7 @@ const AddContributorModal = ({
           ? t('contributors.create_new_author')
           : initialSearchTerm
           ? t('contributors.verify_person')
-          : contributorRole === ContributorRole.EDITOR
-          ? t('contributors.add_editor')
-          : t('contributors.add_author')
+          : getAddContributorText(contributorRole)
```

Comment:
blir ikke dette feil for typer som skal støtte 2+ typer contributors, hvor det vil bli samme tekst uansett om man har trykket "Legg til forfatter" eller "Legg til veileder"?

Reply:
Kommer an på om det skal være to forskjellige moduler eller ikke det da. Du får nesten se hva som er lettest når du skal legge til støtte for flere typer contributors. Denne PRen legger kun til supervisors, ikke flere enn en contributor.